spec/controllers/state_file/questions/md_review_controller_spec.rb

```diff
@@ -1,17 +1,109 @@
-  let(:intake) { create :state_file_md_intake }
+  let(:intake) { create :state_file_md_intake}
 
   before do
     sign_in intake
   end
-
   describe "#edit" do
     render_views
-    it 'succeeds' do
+
+    it "renders" do
       get :edit
       expect(response).to be_successful
     end
+
+    it "shows detailed return information" do
+      intake.direct_file_data.fed_agi = 22_112
+      intake.direct_file_data.fed_taxable_ssb = 1_000
+      intake.direct_file_data.fed_taxable_income = 3_000
+      intake.calculator.lines[:MD502_DEDUCTION_METHOD].instance_variable_set(:@value, "S")
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_3).and_return(3_333)
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_10a).and_return(123)
+      allow_any_instance_of(Efile::Md::Md502SuCalculator).to receive(:calculate_line_u).and_return(85)
+      allow_any_instance_of(Efile::Md::Md502SuCalculator).to receive(:calculate_line_v).and_return(144)
+      allow_any_instance_of(Efile::Md::Md502SuCalculator).to receive(:calculate_line_ab).and_return(42)
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_16).and_return(19_026)
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_17).and_return(1_117)
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_19).and_return(2_987)
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_20).and_return(21_456)
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_21).and_return(2_488)
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_22).and_return(874)
+      allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_line_23).and_return(454)
+      allow_any_instance_of(Efile::Md::Md502crCalculator).to receive(:calculate_md502_cr_part_b_line_4).and_return(512)
+      allow_any_instance_of(Efile::Md::Md502crCalculator).to receive(:calculate_md502_cr_part_m_line_1).and_return(449)
+
+
+      intake.update!(raw_direct_file_data: intake.direct_file_data.to_s)
+
+      get :edit
+
+      page_content = response.body
+
+      expect(page_content).to include I18n.t("state_file.general.see_detailed_return")
+      expect(page_content).to include I18n.t("state_file.general.md_return_type")
+      expect(page_content).to include I18n.t("state_file.general.md_type_of_return_standard")
+      expect(page_content).to include I18n.t("state_file.general.fed_agi")
+      expect(page_content).to include "$22,112"
+      expect(page_content).to include I18n.t("state_file.general.md_state_retirement_pickup_addition")
+      expect(page_content).to include "$3,333"
+      # TODO: add this once lines 9 are implemented
+      # expect(page_content).to include I18n.t("state_file.general.md_subtraction_child_dependent_care_expenses")
```

Comment:
i would actually argue to uncomment these expectations.

```expect(page_content).to include I18n.t("state_file.general.md_subtraction_child_dependent_care_expenses")
```
the translations will exist regardless of the calculator implementation. there isn't any conditional logic surrounding the translations.